server/controllers/adminCtrl.js

```diff
@@ -125,11 +126,18 @@ async function getIndex(req, res, next) {
     if (!memoId) {
       /** ------- Got course code -> prepare course data from kopps for Page 1  ------- */
       log.debug(' getIndex, get course data for : ', { id: thisId })
+
       const { body, statusCode, statusMessage } = await getKoppsCourseData(courseCode, lang)
+      const ladokData = await getLadokCourseData(courseCode, lang)
+      const ladokCourseRounds = await getCourseRoundsData(courseCode, lang)
+      const courseData = { ladokCourseRounds, ladokData, body }
```

Comment:
Is `body` (result from getKoppsCourseData) used anywhere now? It looks like it isn't, so maybe call to getKoppsCourseData can be removed.

Reply:
We also need to look into the other use of `getKoppsCourseData`. It is exposed in server.js as an api route (/api/memo-admin/getKoppsCourseDataByCourse). What I can find it is only used in [handleBack in AdminPage.js](https://github.com/KTH/kurs-pm-admin-web/blob/93a969fd834afbc0fa9ac5f9eb4e0ca0515d502f/public/js/app/views/AdminPage.jsx#L215), but I don't understand why. 
So we need to understand if its needed and then either
- remove it
- or replace it with call to ladok as well